schema/rdl_schema_0.1.json

```diff
@@ -1710,13 +1552,502 @@
+            "ETC",
+            "TCY"
+          ]
+        },
+        "hazard_analysis_type": {
+          "title": "Hazard analysis type",
+          "type": "string",
+          "description": "The type of analysis applied to the hazard data used in the modelled scenario(s), from the closed [analysis type codelist]https://rdl-standard.readthedocs.io/en/{{version}}/data_model/codelists.html#analysis_type).",
+          "codelist": "analysis_type.csv",
+          "openCodelist": false,
+          "enum": [
+            "deterministic",
+            "empirical",
+            "probabilistic"
+          ]
+        },
+        "intensity": {
+          "title": "Hazard intensity measurement",
+          "description": "The metric and units the hazard intensity measurement is given in, from the open [IMT codelist]https://rdl-standard.readthedocs.io/en/{{version}}/data_model/codelists.html#IMT).",
+          "type": "string",
+          "codelist": "IMT.csv",
+          "openCodelist": true,
+          "minLength": 1
+        },
+        "category": {
+          "title": "Exposure category",
+          "description": "The category of the exposed assets, from the closed [exposure category codelist](https://rdl-standard.readthedocs.io/en/{{version}}/data_model/codelists/#exposure_category).",
+          "type": "string",
+          "codelist": "exposure_category.csv",
+          "openCodelist": false,
+          "enum": [
+            "Agriculture",
+            "Buildings",
+            "Infrastructures",
+            "Population",
+            "Natural environment"
+          ]
+        },
+        "cost": {
+          "title": "Asset cost",
+          "type": "array",
+          "description": "The exposure costs associated with specific elements of assets detailed in the dataset.",
+          "items": {
+            "$ref": "#/$defs/Cost"
+          },
+          "minItems": 1,
+          "uniqueItems": true
+        },
+        "taxonomy": {
+          "title": "Exposure taxonomy scheme",
+          "type": "string",
+          "description": "The name of the taxonomy scheme used to create descriptive individual asset feature strings within the dataset. Use of GED4ALL is recommended.",
+          "minLength": 1
+        },
+        "impact": {
+          "title": "Vulnerability impact",
+          "type": "object",
+          "description": "The details of the vulnerability impact values produced in the modelled scenario(s).",
+          "$ref": "#/$defs/Impact"
+        },
+        "spatial": {
+          "title": "Vulnerability spatial details",
+          "description": "Spatial details of the dataset. `.scale` must be provided, other properties of `Location` can be included if they are different from those already detailed in the top level `spatial` object.",
+          "$ref": "#/$defs/Location",
+          "required": [
+            "scale"
+          ]
+        },
+        "functions": {
+          "title": "Vulnerability functions",
```

Comment:
Since only one of the properties is a vulnerability function:

```suggestion
          "title": "Functions",
```

Reply:
Actually, looking at the titles of the properties in `Vulnerability_function`, `Fragility_function`, `Damage_to_loss_function` and `Engineering_demand_function`, should this be "Impact functions"?